sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveSerDeSuite.scala

```diff
@@ -25,17 +25,24 @@ import org.apache.spark.sql.hive.test.TestHive
  * A set of tests that validates support for Hive SerDe.
  */
 class HiveSerDeSuite extends HiveComparisonTest with BeforeAndAfterAll {
-
   override def beforeAll() = {
+    import TestHive._
+    import org.apache.hadoop.hive.serde2.RegexSerDe
     TestHive.cacheTables = false
+    sql(s"""CREATE TABLE IF NOT EXISTS sales (key STRING, value INT)
+       |ROW FORMAT SERDE '${classOf[RegexSerDe].getCanonicalName}'
+       |WITH SERDEPROPERTIES ("input.regex" = "([^ ]*)\t([^ ]*)")
+       """.stripMargin)
+    sql(s"LOAD DATA LOCAL INPATH '${getHiveFile("data/files/sales.txt")}' INTO TABLE sales")
```

Comment:
Thank you @yhuai for noticing this, actually the `createQueryTest` will clean the table by default (as the second argument `reset=true` by default). And that's also why I moved ahead for the test `Read with RegexSerDe`.